.github/workflows/nightly-tests.yml

```
@@ -1,10 +1,24 @@
 name: Nightly Tests
 
 on:
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: 'stage to run test'
```

Update description to match argument name
```suggestion
        description: 'environment to run test in'
```